src/nvim/api/private/helpers.c

```diff
@@ -1034,8 +1038,57 @@ bool extmark_is_range_extremity(Object id) {
   return false;
 }
 
-// Return true if the position is valid TODO
+// Return true if the position is valid TODO(timeyyy): implement
 bool extmark_is_valid_pos(Object id) {
   return true;
 }
 
+// Extmarks may be queried from position or name or even special names
+// in the future such as "cursor". This macro sets the line and col
+// to make the extmark functions recognize what's required
+//
+// *lnum: linenr_T, lnum to be set
+// *col: colnr_T, col to be set
+bool set_extmark_index_from_obj(Buffer buffer, Integer namespace,
+                                Object obj, linenr_T *lnum, colnr_T *col,
+                                Error *err)
+{
+  bool set = false;
```

Comment:
this bool serves no purpose, just `return true` and `return false` directly in the code.